CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Optimizely Python SDK Changelog
 
+## 3.6.0-beta
+September 21st, 2020
+
+### New Features:
+* Version targeting using semantic version syntax [#293](https://github.com/optimizely/python-sdk/pull/293)
+* Datafile accessor [#283](https://github.com/optimizely/python-sdk/pull/283)
```

Comment:
This is not very descriptive. May be `Datafile accessor API to get datafile corresponding to the instance.`